spec/views/tags/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "tags/edit.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
hmm.... i think you forgot to delete this file?

Reply:
yup